assets/style.css

```diff
@@ -0,0 +1,500 @@
+  font-size: 4.5rem;
+  text-transform: uppercase;
+  text-align: center;
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  animation-name: coloranimation;
+  animation-duration: 4s;
+  animation-iteration-count: infinite;
+}
+
+/* SECTION 3: About me / img  */
+
+.about-me-img-container {
+  margin: 1.5rem 2rem;
+
+  display: flex;
+  flex-direction: row;
+}
+
+.img-section {
+  width: 25rem;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.img-section img {
+  width: 100%;
+  height: 100%;
+  object-fit: contain;
+}
+
+.about-me {
+  flex: 2;
+  position: relative;
+  padding-left: 1.5rem;
+}
+
+.about-me-header {
+  font-size: 4.5rem;
+  text-decoration: underline;
+  left: 10%;
+}
+
+.about-me-content {
+  margin: 0;
+  font-size: 1.95rem;
+  left: 10%;
+  top: 20%;
+  text-overflow: inherit;
+}
+
+/* SECTION 4: Main Project */
+
+.entire-project {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.project-header {
+  font-size: 4.5rem;
+  text-decoration: underline;
+  text-align: center;
+  margin-bottom: 2rem;
+}
+
+.main-project {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  position: relative;
+  width: 67rem;
+  border: 0.5rem var(--indigo-dye) solid;
+}
+
+.main-img-container {
+  width: 65rem;
+}
+
+.main-img-container img {
+  width: 100%;
+  height: 100%;
+  object-fit: contain;
+}
+
+.project-title {
+  font-size: 2rem;
+  background-color: var(--light-blue);
+  color: var(--indigo-dye);
+  display: flex;
+  flex-wrap: wrap;
+  width: 20rem;
+  position: absolute;
+  text-align: center;
+  padding: 1.1rem;
+}
+
+.main-project:hover {
+  transform: scale(1.1);
+  opacity: 1;
+}
+
+/* SECTION 5: Future Projects */
+
+.future-projects-container,
+.future-projects {
+  font-size: 2rem;
+  margin-top: 2rem;
+}
+
+.future-projects-container {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
+  margin: 3rem 7rem;
+}
+
+.future-projects {
+  border: 0.5rem var(--indigo-dye) solid;
+  margin-top: 0;
+  margin-bottom: 2rem;
+  width: 30rem;
+  height: 30rem;
+  position: relative;
+}
+
+.future-projects img {
+  width: 100%;
+  height: 100%;
+  object-fit: contain;
+}
+
+.pending-project-title {
+  font-size: 2rem;
+  background-color: var(--light-blue);
+  color: var(--indigo-dye);
+  position: absolute;
+  top: 0;
+  right: 17.5%;
+  text-align: center;
+}
+
+.future-projects:hover {
+  transform: scale(1.1);
+}
+
+/* SECTION 5: contact me  */
+
+.contact-me-container {
+  background-color: var(--indigo-dye);
+}
+
+.contact-me-title {
+  font-size: 4.5rem;
+  text-decoration: underline;
+  text-align: center;
+  margin-bottom: 1rem;
+  color: var(--powder-blue);
+}
+
+.contact-link-container {
+  display: flex;
+  justify-content: space-between;
+  flex-wrap: wrap;
+  align-content: center;
+  margin: 0 3rem;
+  color: var(--powder-blue);
+}
+
+.details {
+  font-size: 2rem;
+}
+
```

Comment:
Same with the title above, some of this falls off the screen when it's past a certain size. Consider making this a bit smaller to make sure it all fits on seamlessly.